enterprise/cloud.oracle/nbproject/project.properties

```diff
@@ -0,0 +1,36 @@
+release.external/bcpkix-jdk15on-1.68.jar=modules/ext/bcpkix-jdk15on-1.68.jar
+release.external/bcprov-jdk15on-1.68.jar=modules/ext/bcprov-jdk15on-1.68.jar
+release.external/commons-codec-1.15.jar=modules/ext/commons-codec-1.15.jar
+release.external/commons-lang3-3.8.1.jar=modules/ext/commons-lang3-3.8.1.jar
+release.external/commons-logging-1.2.jar=modules/ext/commons-logging-1.2.jar
+release.external/hk2-api-2.6.1.jar=modules/ext/hk2-api-2.6.1.jar
+release.external/hk2-locator-2.6.1.jar=modules/ext/hk2-locator-2.6.1.jar
+release.external/hk2-utils-2.6.1.jar=modules/ext/hk2-utils-2.6.1.jar
+release.external/j2objc-annotations-1.3.jar=modules/ext/j2objc-annotations-1.3.jar
+release.external/jackson-annotations-2.12.0.jar=modules/ext/jackson-annotations-2.12.0.jar
+release.external/jackson-core-2.12.0.jar=modules/ext/jackson-core-2.12.0.jar
+release.external/jackson-databind-2.12.0.jar=modules/ext/jackson-databind-2.12.0.jar
+release.external/jackson-datatype-jsr310-2.12.0.jar=modules/ext/jackson-datatype-jsr310-2.12.0.jar
+release.external/jakarta.activation-api-1.2.1.jar=modules/ext/jakarta.activation-api-1.2.1.jar
+release.external/jakarta.annotation-api-1.3.5.jar=modules/ext/jakarta.annotation-api-1.3.5.jar
+release.external/jakarta.inject-2.6.1.jar=modules/ext/jakarta.inject-2.6.1.jar
+release.external/jakarta.ws.rs-api-2.1.6.jar=modules/ext/jakarta.ws.rs-api-2.1.6.jar
+release.external/jersey-apache-connector-2.34.jar=modules/ext/jersey-apache-connector-2.34.jar
+release.external/jersey-client-2.34.jar=modules/ext/jersey-client-2.34.jar
+release.external/jersey-common-2.34.jar=modules/ext/jersey-common-2.34.jar
+release.external/jersey-entity-filtering-2.34.jar=modules/ext/jersey-entity-filtering-2.34.jar
+release.external/jersey-hk2-2.34.jar=modules/ext/jersey-hk2-2.34.jar
+release.external/jersey-media-json-jackson-2.34.jar=modules/ext/jersey-media-json-jackson-2.34.jar
+release.external/jsr305-3.0.2.jar=modules/ext/jsr305-3.0.2.jar
+release.external/nimbus-jose-jwt-9.11.1.jar=modules/ext/nimbus-jose-jwt-9.11.1.jar
+release.external/oci-java-sdk-circuitbreaker-2.10.0.jar=modules/ext/oci-java-sdk-circuitbreaker-2.10.0.jar
+release.external/oci-java-sdk-common-2.10.0.jar=modules/ext/oci-java-sdk-common-2.10.0.jar
+release.external/oci-java-sdk-database-2.10.0.jar=modules/ext/oci-java-sdk-database-2.10.0.jar
+release.external/oci-java-sdk-identity-2.10.0.jar=modules/ext/oci-java-sdk-identity-2.10.0.jar
+release.external/oci-java-sdk-workrequests-2.10.0.jar=modules/ext/oci-java-sdk-workrequests-2.10.0.jar
+release.external/resilience4j-circuitbreaker-1.2.0.jar=modules/ext/resilience4j-circuitbreaker-1.2.0.jar
```

Comment:
I believe these libraries will need `licenseinfo.xml` in the module - CCing @matthiasblaesing  - again I am surprised, no build fails.